pkg/configs/configs.go

```diff
@@ -78,7 +77,8 @@ func GetBaseConfigsFromConfigMap(ctx context.Context, logger logr.Logger, k8sCli
 		configMap := &corev1.ConfigMap{}
 		err := k8sClient.Get(ctx, client.ObjectKey{Namespace: extensionConfigMapNamespace, Name: consts.ConfigMapName}, configMap)
 		if err != nil {
-			return nil, fmt.Errorf("configs: cannot read base configs: %s (%s/%s)", err, extensionConfigMapNamespace, consts.ConfigMapName)
+			logger.Info("configs: cannot configMap return empty baseConfigs")
```

Comment:
I don't understand this message.

Reply:
yes - it is a garbage - that is why no one should commit at 2am :) 

Fixed to something like this:
```
			logger.Info("configs: cannot find configMap with base Configs, return empty map")
```